code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -0,0 +1,95 @@
+/*
+//////////////////////////////////////
+Sensory-Restoration
+	Very very very very noticable.
+	Lowers resistance tremendously.
+	Decreases stage speed tremendously.
+	Decreases transmittablity tremendously.
+	Fatal.
+Bonus
+	The body generates Sensory restorational chemicals.
+	inacusiate for ears
+	antihol for removal of alcohol
+	synaptizine to purge sensory hallucigens
+	mannitol to kickstart the mind
+
+//////////////////////////////////////
+*/
+/datum/symptom/sensory_restoration
+	name = "Sensory Restoration"
+	stealth = -1
+	resistance = -4
+	stage_speed = -4
+	transmittable = -3
+	level = 6
+	severity = 0
+
+/datum/symptom/sensory_restoration/Activate(var/datum/disease/advance/A)
+	..()
+	if(prob(SYMPTOM_ACTIVATION_PROB * 3))
+		var/mob/living/M = A.affected_mob
+		switch(A.stage)
+			if(2)
+				if(M.reagents.get_reagent_amount("inacusiate")<10)
+					M.reagents.add_reagent("inacusiate"=10)
+					M << "<span class='notice'><b>Your hearing feels clearer and crisp</b></span>"
+			if(3)
+				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 )
+					M.reagents.add_reagent_list(list("antihol"=10, "inacusiate"=10))
+					M << "<span class='notice'><b>You feel sober.</b></span>"
+			if(4)
+				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 && M.reagents.get_reagent_amount("synaptizine") < 10)
+					M.reagents.add_reagent_list(list("antihol"=10, "inacusiate"=10, "synaptizine"=5))
```

Comment:
not quite sure if synap is OP here, a constant pump of stun-reducing drugs sounds a bit strong